GameServer/Logic/GameObjects/Nexus.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using LeagueSandbox.GameServer.Logic.Maps;
+using LeagueSandbox.GameServer.Logic.Enet;
+using LeagueSandbox.GameServer.Core.Logic;
+using LeagueSandbox.GameServer.Logic.Packets;
+
+namespace LeagueSandbox.GameServer.Logic.GameObjects
+{
+    public class Nexus : Unit
+    {
+        public Nexus(Map map, uint id, string model, TeamId team, int collisionRadius = 40, float x = 0, float y = 0, int visionRadius = 0) : base(map, id, model, new MinionStats(), collisionRadius, x, y, visionRadius)
```

Comment:
This initialization method is as clusterfuck and should honestly not take in this many arguments. Isn't your doing here, though, as the base requires said values as well.